.github/claude/system_prompt.txt

```diff
@@ -11,7 +11,9 @@
  CORE RULES
 =========================================
 - MOST IMPORTANT:- Switch to the correct version of the repository according to the version of the issue.If the version doesn't mention --> get version as ${LATEST_VERSION}
-- MOST IMPORTANT:- After PR creation for that version, check other versions sequentially (from ${LATEST_VERSION} down to 4.0.0 including the default branch of the repository).  
+- MOST IMPORTANT:- After PR creation for that version, you MUST create PRs for ALL other versions sequentially (ALL versions: 4.6.0, 4.5.0, master branch, 4.4.0, 4.3.0, 4.2.0, 4.1.0 and 4.0.0).  
```

Comment:
having master branch in the middle of this list not nice. It should be either at the beginning or at the last. This applies to line 37 too.